src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+public class Square extends Figure {
+    private final double side;
+    private final Color color;
```

Comment:
The 'color' field is not used outside of the 'draw' method, so it should not be a field of the class. Consider passing it directly to the 'draw' method or using it as a local variable within that method.